README.md

```diff
@@ -34,7 +34,7 @@ number of dependencies for installation. For full details please see [INSTALL.md
 We're always keen to have more developers! Pull requests are very welcome.
 
 * Bugs are recorded in the [issue tracker](https://github.com/openstreetmap/openstreetmap-website/issues).
-* Translation is managed by [Translatewiki](https://translatewiki.net/wiki/Translating:OpenStreetMap).
+* Translation is managed by [Translatewiki](https://translatewiki.net/wiki/Translating:OpenStreetMap), for communities's name we use data in [OCI](https://github.com/osmlab/osm-community-index) and work on [Transifex](https://www.transifex.com/openstreetmap/id-editor/languages/)
```

Comment:
I'm not entirely sure we want to complicate things like this for an edge case but it we do merge this then it should be `community names` not `communities's name`.

Reply:
> I'm not entirely sure we want to complicate things like this for an edge case but it we do merge this then it should be `community names` not `communities's name`.

Ah, I didn't realize the plural should not be used here, thanks for the English grammar correction